### ISSUE TITLE
https://internxt.com has 34 trackers

### ISSUE THREAD

**Issue by candroid-man** [internxt/X-Cloud-Web#384]
uBlock Origin detects 34 trackers on your website including Facebook and Google, and for being a privacy respecting service, that is not a good look. The Android app also has 2 trackers from Segment and Google.

Website trackers:

ads-twitter.com
d1tcqh4bio8cty.cloudfront.net
m.stripe.network
facebook.net
google-analytics.com
googletagmanager.com
hotjar.com
intercom.io
mailerlite.com
redditstatic.com
segment.com
stripe.com
stripe.network
stripecdn.map.fastly.net

**Comment by schnoog** [comment]
@fvsegarra   co: @sg-gs 

Why was this issue closed but the trackers are still active (most of them)?
Where can I find this in the ToC?
What's the "technical justificaton" for having f**king Reddit tracking us while using the drive?

**_GDPR?_**

That's the second massive fail of internxt (after the cli webdav on 0.0.0.0 by default). 
I'm more than just a little bit angry and thinking about raising an official GDRP complaint with your government authority. 

**I expect a timely answer.**

**Comment by fvsegarra** [comment]
Hello, Thanks for bringing this up.
We use non-invasive pixels or cookies on our website, for instance, to measure website views, as specified in our Terms of Service and Privacy Policy. These do not make Internxt Drive any less secure. Litearlly all websites, including those of our competitors, use some sort of pixel code on their website; either to measure website visitors, purchases, or whatever

**Comment by schnoog** [comment]
Please show me where in your ToS I can find that
Reddit 
and 
Facebook 
are used.

Where did I gave my consent, as required by Article 6 of the GDPR for non essential parts of the services?


"Litearlly all websites, including those of our "
Yes, and now show me who of those "all website" tell us about privacy by default.

**Comment by fvsegarra** [comment]
<img width="583" height="363" alt="Image" src="https://github.com/user-attachments/assets/fb181bf0-8644-4054-8f80-5c664e208922" />
<img width="866" height="373" alt="Image" src="https://github.com/user-attachments/assets/cc74e535-4146-43b0-a8db-96f02dd0d925" />

**Comment by fvsegarra** [comment]
Also, please note that pretty much all those pixels from back in 2022 are no longer active and have all been removed

**Comment by schnoog** [comment]
Cool....

Some examples from some minutes ago:

facebook.net -> https://connect.facebook.net/en_US/fbevents.js
newrelic.com -> https://js-agent.newrelic.com/nr-rum-1.250.0.min.js
redditstatic.com -> https://www.redditstatic.com/ads/pixel.js

So an active request from my computer is made towards reddit, which in returns has my browser fingerprint, my ip address (IPv6 even endpoint) and can process it. That all is personal data according to the GDPR.

And sorry, you talk about post quantum encryption but ignore the risk of supply chain attacks in regards to third party hosted JS (for which you not even implemented a checksum test)? 

I mean really. Just make it public. Just write right below your "privacy by default" that "we inform Reddit, Facebook and others about your visit on your paid drive and include active code from them which technically has the means to read your user input, including passwords"

**Comment by candroid-man** [comment]
I agree, this is beyond shady for a company that claims to care about user privacy, this is NOT what privacy looks like.
